core/src/main/scala/neotypes/implicits/mappers/ResultMappers.scala

```diff
@@ -4,15 +4,15 @@ package implicits.mappers
 import java.time._
 import java.util.UUID
 
-import mappers.{ResultMapper, TypeHint, ValueMapper}
-import types.Path
-
+import neotypes.exceptions.MultipleIncoercibleException
+import neotypes.mappers.{ResultMapper, TypeHint, ValueMapper}
+import neotypes.types.Path
```

Comment:
All over the code base, we have been using the convention of using relative imports for importing things of the same project.
We have also kept them separated from other imports to make it clear to the eyesight.